webapp/app/js/perf.js

```diff
@@ -82,6 +82,14 @@ perf.factory('PhSeries', ['$http', 'thServiceDomain', function($http, thServiceD
           });
 
         if (seriesSummary) {
+          if (!seriesSummary.subtestSignatures) {
+            seriesSummary = _.find(lists.seriesList,
```

Comment:
It occurs to me that this won't work if we have multiple suites with the same subtests. Not currently the case, but it could happen if we changed a suite. So we should allow for multiple results here (and getting the union of all subtest signatures from them).